src/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { StrictMode } from 'react';
+import ReactDOM from 'react-dom';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import HomePage from '/elements/HomePage';
+import CountryList from '/elements/CountryList';
+import Population from '/elements/Population';
+import Chart from '/elements/Chart';
+
+// import { createRoot } from 'react-dom/client';
+//import App from './App';
+
+const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <HomePage />,
+  },
+  {
+    path: '/list',
+    element: <CountryList />,
+  },
+  {
+    path: '/population',
+    element: <Population />,
+  },
+  {
+    path: '/chart',
+    element: <Chart />,
+  },
+]);
```

Comment:
Close! Your paths almost correct, you've got the strings on the opposite side of the slash, see mine below.

```
const router = createBrowserRouter([
  {
    path: "/",
    element: <Navigate to="home/" />,
  },
  {
    path: "home/",
    element: <Home />,
  },
  {
    path: "list/",
    element: <List />,
  },
  {
    path: "population/",
    element: <Population />,
  },
  {
    path: "customroute/",
    element: <CustomRoute />,
  },
]);
```